lib/api_client.js

```diff
@@ -2090,7 +2094,8 @@ APIClient.prototype.sendTransaction = function(identifier, txHex, paths, checkFe
         "/wallet/" + identifier + "/send",
         {
             check_fee: checkFee ? 1 : 0,
-            prioboost: prioboost ? 1 : 0
+            prioboost: prioboost ? 1 : 0,
+            bip70: options.bip70
```

Comment:
Could you name this guy something different? 'bip70PaymentUrl' 'submitPaymentMessageUrl' or something. bip70 might be a tiny bit too vague :)